docs/ref/paginator.txt

```diff
@@ -161,6 +161,73 @@ Attributes
 
     A 1-based range iterator of page numbers, e.g. yielding ``[1, 2, 3, 4]``.
 
+``AsyncPaginator`` class
+========================
+
+.. class:: AsyncPaginator(object_list, per_page, orphans=0, allow_empty_first_page=True, error_messages=None)
```

Comment:
Please add versionadded annotations.

Reply:
I missed that! Thanks for letting me know😀